src/senaite/core/setuphandlers.py

```diff
@@ -221,6 +227,10 @@ def add_dexterity_setup_items(portal):
     setup = api.get_setup()
     add_dexterity_items(setup, items)
 
+    # disable snapshots for setup folders
+    objects = [setup.get(item[0]) for item in items]
+    map(disable_snapshots, objects)
```

Comment:
Why is this required when the base class already implements `IDoNotSupportSnapshots`?

Reply:
Good point

Reply:
Removed with https://github.com/senaite/senaite.core/pull/2479/commits/94ff2a92c1be84c8c433bad3e53216c150044de4